exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/window/TestWindowFrame.java

```diff
@@ -106,6 +106,60 @@ public void testMultipleFramers() throws Exception {
     );
   }
 
+  @Test
+  public void testUnboundedFollowing() throws Exception {
+    testBuilder()
+      .sqlQuery(getFile("window/q3.sql"), TEST_RES_PATH)
+      .ordered()
+      .sqlBaselineQuery(getFile("window/q4.sql"), TEST_RES_PATH)
+      .build()
+      .run();
+  }
+
+  @Test
+  public void testAggregateRowsUnboundedAndCurrentRow() throws Exception {
+    final String table = "dfs_test.`"+TEST_RES_PATH+"/window/b4.p4`";
+    testBuilder()
+      .sqlQuery(getFile("window/aggregate_rows_unbounded_current.sql"), table)
+      .ordered()
+      .sqlBaselineQuery(getFile("window/aggregate_rows_unbounded_current_baseline.sql"), table)
+      .build()
+      .run();
+  }
+
+  @Test
+  public void testLastValueRowsUnboundedAndCurrentRow() throws Exception {
+    final String table = "dfs_test.`"+TEST_RES_PATH+"/window/b4.p4`";
+    testBuilder()
+      .sqlQuery(getFile("window/last_value_rows_unbounded_current.sql"), table)
+      .unOrdered()
+      .sqlBaselineQuery(getFile("window/last_value_rows_unbounded_current_baseline.sql"), table)
+      .build()
+      .run();
+  }
+
+  @Test
+  public void testAggregateRangeCurrentAndCurrent() throws Exception {
```

Comment:
the added tests use the window/b4.p4 dataset that contains 4 partitions. The second partition has a frame that has 3 rows at the end of the 1st batch and 2 rows at the beginning of the second batch.
Is this enough, or should I generate another dataset with a frame that spans over more than 2 batches ?